test/static_test/test_base64.cc

```diff
@@ -3,7 +3,7 @@
 
 #include "gtest/gtest.h"
 #include "test_base64.h"
-#include "base64.h"
+#include "utils/base64.h"
```

Comment:
keep where it was, unless the class is really be reused.
As we discussed before, the base64 is unnecessary and is harmful for mobile product.

Reply:
Sure, I will move back them.

Reply:
I delete it because base64 isn't used in our project.

Reply:
I think we can delete base64, the only usage in our project is here
```c++
  if (base64_decode(model_as_string, model_as_bytes)) {
    model_proto.ParseFromArray(model_as_bytes.data(), model_as_bytes.size());
  } else {
    model_proto.ParseFromArray(model_as_string.c_str(), model_as_string.size());
  }
```